class.js

```diff
@@ -1,20 +1,23 @@
 class Foo
 {
-color='negro';peso   =    34;
+  color = 'negro';
+  peso = 34;
+  
   ver()
   {
```

Comment:
La llave se abre en la línea misma.
```suggestion
  ver() {
```

Reply:
Aplicar donde corresponda.